### ISSUE TITLE
Training loss goes to zero within 100 training steps

### ISSUE THREAD

**Issue by anumita0203** [mingzhu0527/MASHQA#5]
When training the model using the default settings the training loss goes to zero within 100 training steps. I'm using tensorflow-gpu 1.14. I've verified that the input data is read correctly.

Can you tell me how to fix this issue?
